File-Formats/Presentation/Presentation-to-PDF.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Converting PowerPoint Presentation to PDF |Syncfusion|
-description: This section illustrates how to convert PowerPoint Presentation document to PDF; PowerPoint Presentation conversion
+title: Converting Presentation to PDF in File Formats Presentation|Syncfusion
```

Comment:
.NET PowerPoint Framework - Convert PowerPoint to PDF | Syncfusion

Reply:
resolved

Reply:
resolved